config/routes.rb

```diff
@@ -116,7 +116,9 @@
     resource :choose_programme, controller: :choose_programme, only: %i[show create], path: "choose-programme"
     resources :cohorts do
       member do
-        get "/:start_year", action: :show
+        get "show"
```

Comment:
This one you probably don't need to define. When you have a resource like cohorts above rails automaticallys creates /cohorts/:id and you can use that for the show